### PR TITLE
Prod `countAdvertisers` cloud functions

### PR DESCRIPTION
**Pull request by rob-marcus**
Title. Just a duplicate of the `dev` code with various constants updated.  

Re the force-push: 
I accidentally responded to scarlets comments in the wrong file : ( and pushed. Had to reset with a force push to revert.

### CODE REVIEW

File: server/utils/cloudFunctions/functions/countAdvertisersCloudFunctions/countAdvertisersConfig.js [hunk 1]
```diff
@@ -12,6 +12,10 @@ const DEV_AGGREGATES_COLLECTION_NAME = "dev_aggregates";
 const DEV_AGGREGATES_COLLECTION = 
     DB.collection(DEV_AGGREGATES_COLLECTION_NAME);
 
+const PROD_AGGREGATES_COLLECTION_NAME = "aggregates";
+const PROD_AGGREGATES_COLLECTION =
```
Comment (scarletnguyen13) [inline server/utils/cloudFunctions/functions/countAdvertisersCloudFunctions/countAdvertisersConfig.js:16]: nit: have this one on the same line maybe?

File: server/utils/cloudFunctions/functions/countAdvertisersCloudFunctions/prodCountAdvertisersFunctions.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    require('./countAdvertisersHelpers');
+
+/**
+ * Increments an advertiser's annual ad count by one for an ad in a given year.
+ * in prod env. Annual ad count stored in firebase collection `PROD_AGGREGATES_COLLECTION`.
+ * Event triggered on create.
+ * @returns {!Promise}     
+ */
+exports.prodCountAdvertisersOnCreate = 
+  PROD_ADS_DOCS.onCreate(snapshot => {
```
Comment (scarletnguyen13) [inline server/utils/cloudFunctions/functions/countAdvertisersCloudFunctions/prodCountAdvertisersFunctions.js:34]: just `snapshot => incrementAdvertiserCount(snapshot, PROD_AGGREGATES_COLLECTION)` is good enough

File: server/utils/cloudFunctions/functions/countAdvertisersCloudFunctions/prodCountAdvertisersFunctions.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    });
+
+/**
+ * Decrements an advertiser's annual ad count by one for an ad in a given year.
+ * in prod env. Annual ad count stored in firebase collection `PROD_AGGREGATES_COLLECTION`.
+ * Event triggered on delete.
+ * @returns {!Promise} 
+ */
+exports.prodCountAdvertisersOnDelete = 
+    PROD_ADS_DOCS.onDelete(snapshot => {
```
Comment (scarletnguyen13) [inline server/utils/cloudFunctions/functions/countAdvertisersCloudFunctions/prodCountAdvertisersFunctions.js:45]: Same as above

File: server/utils/cloudFunctions/functions/countAdvertisersCloudFunctions/countAdvertisersConfig.js (hunk 1 above)
Comment (rob-marcus) [reply server/utils/cloudFunctions/functions/countAdvertisersCloudFunctions/countAdvertisersConfig.js:16]: 80 char limit

**Pull request merged**